README.md

```diff
@@ -1 +1,47 @@
-# display-input-data
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# HTML Forms
+
+> This project consists of building an HTML document that matches the appearance of mint.com’s signup page.
+
+![screenshot][screenshot_url]
+
+[screenshot_url]: https://github.com/jrai0792/display-input-data/blob/main/127.0.0.1_60302_(CC's%20Laptop).png?raw=true
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML,
+- CSS
+
+
+## Authors
+
+👤 **Author1**
+
+- Github: [@jrai0792](https://github.com/githubhandle)
+
+👤 **Author2**
+
+- Github: [@oracleot](https://github.com/githubhandle)
+
+## 🤝 Contributing
+
+Contributions, issues and feature requests are welcome!
+
+Feel free to check the [issues page](issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] **[MANDATORY]** This line is also just a placeholder in the template, you should really add acknowledgements for your project if you are going to use this section, you can put odin project page, microverse page, or any other resource you used